reaper/reaper/src/main/kotlin/com/emergetools/reaper/ReaperReportUploadWorker.kt

```diff
@@ -23,12 +23,14 @@ import okhttp3.Request
 import okhttp3.RequestBody.Companion.toRequestBody
 import okhttp3.Response
 import okhttp3.internal.closeQuietly
+import java.io.ByteArrayOutputStream
 import java.io.DataInputStream
 import java.io.EOFException
 import java.io.File
 import java.io.FileInputStream
 import java.io.FileOutputStream
 import java.io.IOException
+import java.util.zip.GZIPOutputStream
 import kotlin.coroutines.resumeWithException
 
 internal class ReaperReportUploadWorker(
```

Comment:
I thought sorting the records might help with compression, and it does, but not as much as I had hoped:

```
$ # Without sorting:
$ python3 -c 'import json; import random; import base64; print(json.dumps([base64.b64encode(random.randbytes(8)).decode("utf8") for i in range(100000)]))' | gzip --verbose >/dev/null
 41.7%
$ # With sorting:
$ python3 -c 'import json; import random; import base64; print(json.dumps(sorted([base64.b64encode(random.randbytes(8)).decode("utf8") for i in range(100000)])))' | gzip --verbose >/dev/null
 46.3%
```
(Number is compression ratio)

So maybe not worth it.